domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt

```diff
@@ -36,6 +37,8 @@ private const val ADD_PROFILE_TRANSFORMED_PROVIDER_ID = "add_profile_transformed
 private const val UPDATE_NAME_TRANSFORMED_PROVIDER_ID = "update_name_transformed_id"
 private const val UPDATE_PIN_TRANSFORMED_PROVIDER_ID = "update_pin_transformed_id"
 private const val UPDATE_DOWNLOAD_ACCESS_TRANSFORMED_PROVIDER_ID = "update_download_access_transformed_id"
+private const val UPDATE_DOWNLOAD_AND__UPDATE_ONLY_ON_WIFI_TRANSFORMED_PROVIDER_ID = "update_download_and_update_only_on_wifi_transformed_id"
```

Comment:
Nit: There is `__` double underscore. Fix that.

Reply:
Done.